proto/github.proto

```diff
@@ -89,6 +89,10 @@ message AppInstallation {
   // The ID of the GitHub app being linked. There are only 2 possible values,
   // corresponding to either the read-write or read-only app.
   int64 app_id = 4;
+
+  // Whether to automatically report GitHub commit statuses for build with
+  // role=CI.
```

Comment:
The UI text says this applies to workflows too (ROLE=CI_RUNNER). Also, does this apply to ROLE=HOSTED_BAZEL?

Reply:
No - we only report statuses for role=CI_RUNNER and CI

Updated to mention CI_RUNNER though